rpcserver.go

```diff
@@ -737,30 +730,22 @@ func (r *rpcServer) addDeps(s *server, macService *macaroons.Service,
 
 	// External subserver possibly need to register their own permissions
 	// and macaroon validator.
-	if r.extSubserverCfg != nil {
-		macValidator := r.extSubserverCfg.MacaroonValidator
-		for method, ops := range r.extSubserverCfg.Permissions {
-			err := r.interceptorChain.AddPermission(method, ops)
-			if err != nil {
-				return err
-			}
+	for method, ops := range r.implCfg.ExternalValidator.Permissions() {
```

Comment:
No, since we expect all interfaces to be non-nil. But the default implementation can just return a nil map of permissions which means we don't add any external validators here.